### PR TITLE
[irrlicht] add feature to support unicode path on windows

### PR DESCRIPTION
**Pull request by DailyShana**
**Describe the pull request**

- What does your PR fix? Fixes issue #
add feature [unicode] to support unicode path on windows
- Which triplets are supported/not supported? Have you updated the CI baseline?
no change
- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?

### CODE REVIEW

**Review by PhoebeHui — CHANGES_REQUESTED** [review]
@DailyShana, thanks for the PR! BTW, have you test the feature locally with x86-windows, x64-windows, and Linux?

**Comment by DailyShana** [comment]
all features tested locally with x86-windows, x64-windows, x64-windows-static
unicode feature is only meaningful on windows, and I have no condition to test other features on linux

**Comment by PhoebeHui** [comment]
@DailyShana, thanks for your updates!

The CI failed on Linux doesn't relate to this changes.

**Review by PhoebeHui — APPROVED** [review]

File: ports/irrlicht/CONTROL [hunk 1]
```diff
@@ -1,9 +1,12 @@
 Source: irrlicht
-Version: 1.8.4-4
+Version: 1.8.4-5
 Homepage: http://irrlicht.sourceforge.net
 Description: Irrlicht lightning fast 3d engine
 Build-Depends: zlib, libpng, bzip2, libjpeg-turbo
 
+Feature: unicode
+Description: Support unicode path on windows
```
Comment (ras0219-msft) [inline ports/irrlicht/CONTROL]: Should this be a default feature?

Comment (DailyShana) [reply ports/irrlicht/CONTROL]: It should be default as long as allowing somewhat incompatibility

**Comment by ras0219-msft** [comment]
LGTM, thanks for the PR!

**Pull request merged**